### ISSUE TITLE
FR: make `brew install kurtosis` a separate box in cloud.kurtosis.com

### ISSUE THREAD

**Issue by leeederek** [kurtosis-tech/kurtosis#1128]
### Background & motivation

<img width="720" alt="image" src="https://github.com/kurtosis-tech/kurtosis/assets/103802618/9e04c0e6-40e2-4a14-9c84-15fc233dd860">

Currently, a new user will get a single box with 3 lines to copy-paste into their terminal. There is added noise when a user who already has Kurtosis installed runs this because they will see a message that reads: `kurtosis-tech/tap/kurtosis-cli 0.81.8 is already installed` and also, this step is unnecessary 

### Desired behaviour

There are 2 boxes, both of which have the copy-paste functionality:

**First box will say:**

If you haven't already, please install the Kurtosis CLI to interact with the cloud enclave:
`brew install kurtosis-tech/tap/kurtosis-cli`
[COPY BUTTON]

**Second box will say:**

Assuming you have kurtosis installed now, use the following two lines to get connected to your cloud instance:
```
$ kurtosis engine start && kurtosis portal start

$ export KURTOSIS_CLOUD_API_KEY="U441y6VcuwqhKLhPwwuXWyQRaHjas5e1qKRZDcpSefk="

$ kurtosis cloud load d67967268c8343818bd7ca2c16952a53
```

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

**Comment by adschwartz** [comment]
@leeederek what do you think if we just link to our installation instructions. That actually seems most appropriate because installing kurtosis-cli involves more than the one liner I added for now.

For context: this installation text is meant as a placeholder until we send them to the enclave manager. I'm not actually sure that @mieubrisse wants these instructions at all.

**Comment by mieubrisse** [comment]
Yep I don't. Interacting with Cloud via the CLI is an advanced usecase (relative to just getting thrown into the enclave manager)
